presto-parser/src/main/java/com/facebook/presto/sql/tree/LambdaArgumentDeclaration.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.sql.tree;
+
+import java.util.Objects;
+import java.util.Optional;
+
+public class LambdaArgumentDeclaration
```

Comment:
Yes, we do. We talked about before. This is needed to make associating symbols and types with lambdas easy. The alternative is to use QualifiedNameReference and SymbolReference here. Using a dedicated types seems preferable here.